### PR TITLE
Julia 1.0 tests, sparse convert, mutating matmul in sparse

### PR DESCRIPTION
**Pull request by dkarrasch**
Not sure about whether this should be 2.2.0 or 2.1.1, but I guess the little additional feature (sparse convert) would justify the former.

### CODE REVIEW

**Comment by Jutho** [comment]
I'll take a look tonight

File: src/composition.jl [hunk 1]
```diff
@@ -87,7 +87,7 @@ function A_mul_B!(y::AbstractVector, A::CompositeMap, x::AbstractVector)
         A_mul_B!(dest, A.maps[1], x)
         source = dest
         if N>2
-            dest = Array{T}(undef, size(A.maps[2],1))
```
Comment (Jutho) [inline src/composition.jl]: There are probably many lines with missing spaces, e.g. line 93 below as well. Not a comment, just a thought.

**Comment by Jutho** [comment]
Look's good to me; I guess you have all the privileges to merge and tag a new release.

**Comment by dkarrasch** [comment]
Thanks @Jutho. I went through the other files as well and made the use of spaces after commas consistent. I'll release a version 2.2.0 then.

**Pull request merged**